DependencyInjection/Configuration.php

```diff
@@ -131,12 +161,13 @@ private function createPluginsNode()
      */
     private function createBindNode()
     {
-        return $this->createNode('bind')
-            ->useAttributeAsKey('name')
-                ->prototype('array')
+        return $this
+            ->createNode('bind')
                 ->useAttributeAsKey('name')
-                ->prototype('variable')->end()
-            ->end();
+                ->prototype('array')
+                    ->useAttributeAsKey('name')
+                    ->prototype('variable')->end()
+                ->end();
```

Comment:
both bind and plugin nodes are not supported by XML now. What do you think of naming them `binds` and `plugins`? (as we can add `fixXmlConfig` for it then)